src/main/scala/exercises/06_Collections.scala

```diff
@@ -31,14 +31,18 @@ object Collections {
    * 
    */
   def firstElementInList[T](l: List[T]): T = {
-    error("fix me")
+    l.head
   }
 
   /**
    * Get the sum of all the elements in the list, e.g. sumOfList(List(1,2,3)) = 6.
    */
   def sumOfList(l: List[Int]): Int = {
```

Comment:
I can't tell which version I like more, yours or mine.  I did:
```scala
 l.foldLeft(0)(_ + _) 
```
Which uses the additive identity to add up every element of the list using the a lambda that sums two values.
Yours is incredibly clear in what it is doing, while mine is very concise.  I'm curious what you think about this?